skimage/feature/texture.py

```diff
@@ -301,7 +301,7 @@ def local_binary_pattern(image, P, R, method='default'):
     .. [1] Multiresolution Gray-Scale and Rotation Invariant Texture
            Classification with Local Binary Patterns.
            Timo Ojala, Matti Pietikainen, Topi Maenpaa.
-           http://www.rafbis.it/biplab15/images/stories/docenti/Danielriccio/Articoliriferimento/LBP.pdf, 2002.
+           http://vision.stanford.edu/teaching/cs231b_spring1415/papers/lbp.pdf, 2002.
```

Comment:
Why not use the official link:
http://www.ee.oulu.fi/research/mvmp/mvg/files/pdf/pdf_94.pdf ?

Reply:
Sure, even better. Should I submit and updated PR?